plugins/openwhisk/src/main/java/com/navercorp/pinpoint/plugin/openwhisk/OpenwhiskPlugin.java

```diff
@@ -0,0 +1,115 @@
+package com.navercorp.pinpoint.plugin.openwhisk;
+
+import com.navercorp.pinpoint.bootstrap.async.AsyncContextAccessor;
+import com.navercorp.pinpoint.bootstrap.instrument.InstrumentClass;
+import com.navercorp.pinpoint.bootstrap.instrument.InstrumentException;
+import com.navercorp.pinpoint.bootstrap.instrument.InstrumentMethod;
+import com.navercorp.pinpoint.bootstrap.instrument.Instrumentor;
+import com.navercorp.pinpoint.bootstrap.instrument.transformer.TransformCallback;
+import com.navercorp.pinpoint.bootstrap.instrument.transformer.TransformTemplate;
+import com.navercorp.pinpoint.bootstrap.instrument.transformer.TransformTemplateAware;
+import com.navercorp.pinpoint.bootstrap.plugin.ProfilerPlugin;
+import com.navercorp.pinpoint.bootstrap.plugin.ProfilerPluginSetupContext;
+import com.navercorp.pinpoint.plugin.openwhisk.accessor.PinpointTraceAccessor;
+
+import java.security.ProtectionDomain;
+import java.util.Optional;
+
+/**
+ * @author Seonghyun Oh
+ */
+public class OpenwhiskPlugin implements ProfilerPlugin, TransformTemplateAware {
+
+    private TransformTemplate transformTemplate;
+
+    @Override
+    public void setup(ProfilerPluginSetupContext context) {
+        final OpenwhiskConfig config = new OpenwhiskConfig(context.getConfig());
+        if (!config.isEnable()) {
+            return;
+        }
+
+        OpenwhiskDetector openwhiskDetector = new OpenwhiskDetector();
+        context.addApplicationTypeDetector(openwhiskDetector);
+
+        transformTemplate.transform("whisk.http.BasicHttpService$$anonfun$assignId$1$$anonfun$apply$13", new TransformCallback() {
```

Comment:
It would be nice if could change the target using settings.
It is good for maintenance.